sdk/core/core-client/src/serviceClient.ts

```diff
@@ -122,8 +135,8 @@ export class ServiceClient {
     operationArguments: OperationArguments,
     operationSpec: OperationSpec
   ): Promise<T> {
-    const baseUri: string | undefined = operationSpec.baseUrl || this._baseUri;
-    if (!baseUri) {
+    const endpoint: string | undefined = operationSpec.baseUrl || this._endpoint;
+    if (!endpoint) {
       throw new Error(
         "If operationSpec.baseUrl is not specified, then the ServiceClient must have a baseUri string property that contains the base URL to use."
```

Comment:
update this error to mention `endpoint` ?

Reply:
updated.